web/styles/app_components.css

```diff
@@ -1221,14 +1222,18 @@ div.overlay {
         align-items: center;
         box-sizing: border-box;
         width: 100%;
-        height: 100%;
-        margin: 1px;
+        height: 90%;
```

Comment:
I don't understand this `90%` height value. The container is 33px high, so why not just say that the inner tab is 31px high; that coupled with flexbox center properties will leave a 1px space around the buttons without having to specify a margin. Then, instead of left and right margins, just specify a `1px` gap on the `.tabs-container`.

Reply:
Used 31px for height.
1px padding on `.tabs-container` and 1px gap

I think we should keep the gap 2px as the gap is hardly visibile on 100% zoom ?

Reply:
Makes sense to me.